<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Serial Window</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-games">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(21, 21, 21);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="veriGonderGrupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>301</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Veri Gönder</string>
    </property>
    <widget class="QLineEdit" name="veriGonderTextBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	font: 9pt &quot;Consolas&quot;;
border-radius:10x;
	background-color: rgb(255, 255, 255);
	
	color: rgb(139, 139, 139);
 
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Veri giriniz...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="veriGonderButon">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>8</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 700 8pt &quot;Segoe UI&quot;;
	
	
	background-color: rgb(0, 63, 188);
	
    border-radius:10px;
    background-repeat: no-repeat;
}

QPushButton:pressed { 
	background-color: rgb(0, 85, 255);
    border-radius:13px;
    background-repeat: no-repeat;
    border:none;
}</string>
     </property>
     <property name="text">
      <string>   Gönder</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="veriGonderGrupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Bağlantı Ayarları</string>
    </property>
    <widget class="QPushButton" name="seriPortAcButon">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>8</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 700 8pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
	
	background-color: rgb(0, 170, 0);
    border-radius:5px;
    background-repeat: no-repeat;
}

QPushButton:pressed { 
	
	background-color: rgb(0, 255, 0);
    border-radius:7px;
    background-repeat: no-repeat;
    border:none;
}</string>
     </property>
     <property name="text">
      <string>Bağlantı Kur</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="portComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
	font: 9pt &quot;Consolas&quot;;

	background-color: rgb(255, 255, 255);

	color: rgb(139, 139, 139);
 
}
</string>
     </property>
     <property name="placeholderText">
      <string>Port Seçiniz</string>
     </property>
    </widget>
    <widget class="QComboBox" name="baudeRateComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
	font: 9pt &quot;Consolas&quot;;

	background-color: rgb(255, 255, 255);

	color: rgb(139, 139, 139);
 
}
</string>
     </property>
     <property name="placeholderText">
      <string>Bauderate Seçiniz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="seriPortKapatButon">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 700 8pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
	
	background-color: rgb(186, 0, 0);
	   border-radius:5px;
    background-repeat: no-repeat;
}

QPushButton:pressed { 
	background-color: rgb(255, 0, 0);
    border-radius:7px;
    background-repeat: no-repeat;
    border:none;
}</string>
     </property>
     <property name="text">
      <string>Bağlantı Kes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="veriGonderGrupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Okunan Veriler</string>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Gelen Veri 1  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Gelen Veri 2  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Gelen Veri 4  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Gelen Veri 3  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="gelenVeriADC1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 229, 0);</string>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="gelenVeriADC2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 229, 0);</string>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="gelenVeriADC3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 229, 0);</string>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="gelenVeriADC4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 229, 0);</string>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="veriGonderGrupBox_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Bağlantı Bilgileri</string>
    </property>
    <widget class="QLabel" name="dLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 234, 0);</string>
     </property>
     <property name="text">
      <string>Durum :</string>
     </property>
    </widget>
    <widget class="QLabel" name="durumLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 225, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="veriGonderGrupBox_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>301</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Terminal Ekranı</string>
    </property>
    <widget class="QTextEdit" name="terminalEkrani">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
font: 11pt &quot;Consolas&quot;;
color: rgb(0, 255, 0);
border-left-color: rgb(0, 255, 0);
gridline-color: rgb(0, 255, 0);
selection-color: rgb(255, 255, 0);
selection-background-color: rgb(255, 255, 0);</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="terminalEkranitTemizle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 700 8pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
	
	background-color: rgb(255, 170, 0);
    border-radius:10px;
    background-repeat: no-repeat;
}

QPushButton:pressed { 
	
	background-color: rgb(0, 0, 0);
    border-radius:13px;
	color: rgb(255, 255, 255);
    background-repeat: no-repeat;
    border:none;
}</string>
     </property>
     <property name="text">
      <string>Temizle</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>terminalEkrani</tabstop>
  <tabstop>veriGonderButon</tabstop>
  <tabstop>seriPortAcButon</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
